whois-api/src/main/java/net/ripe/db/whois/api/rdap/domain/LinkRelationType.java

```diff
@@ -0,0 +1,32 @@
+package net.ripe.db.whois.api.rdap.domain;
+
+public enum LinkRelationType {
+    UP("rdap-up"),
+    TOP("rdap-top"),
+    DOWN("rdap-down"),
+    BOTTOM("rdap-bottom"),
+    ACTIVE("rdap-active");
+
+    private final String value;
+
+    LinkRelationType(String value) {
+        this.value = value;
+    }
+
+    public String getValue() {
+        return value;
+    }
+
+    public static String concatActiveLinkRelation(final LinkRelationType type) {
+        return type.value.concat(" ").concat(ACTIVE.getValue());
+    }
+
+    public static boolean containsValidValue(final String value) {
+        for (RelationType type : RelationType.values()) {
+            if (value.contains(type.getValue())) {
```

Comment:
Value must be case-sensitive? Does formatting matter? Can value contain multiple values?

Reply:
This is only used for Its and it is always lowercase. So doesn't really matter

Reply:
OK